kubeflow/components/bootleg.yaml

```diff
@@ -0,0 +1,143 @@
+name: Bootleg
+description: |
+  Named Entity Disambiguation
```

Comment:
What is the purpose of this task? I see it's called Bootleg, but actually it trains a semantic parser?
Why can you not accomplish what you want with additional arguments to the regular train component?

Reply:
Agreed, I need to fix this part. This script only runs a bootleg model e2e, produce candidates and features, and cache them in separate files. I need to add a proper genienlp command to call here instead of train.